model/document.js

```diff
@@ -197,7 +197,32 @@ module.exports = memoize(function (db) {
 			if (statusLog.length) data.statusLog = statusLog;
 
 			return data;
-		} }
+		} },
+		toWSSchema: {
+			value: function (ignore) {
+				var schema = {
+					type: "array",
+					items: {
+						type: "object",
+						properties: {
+							code: {
+								type: "enum",
+								ref: "documents"
+							},
+							data: {}
+						}
+					}
+				};
+				schema.items.properties.files =
+					this.files.map._descriptorPrototype_.type.prototype.toWSSchema();
+				schema.items.properties.owner = { type: "string" };
+				if (this.dataForm.constructor !== this.database.FormSectionBase) {
+					schema.items.properties.data = this.dataForm.toWSSchema();
+					schema.items.dataForms = [ { title: "", properties: [] } ];
```

Comment:
@roxanadina Please specify how this should be filled. From [your comment](https://unctad.atlassian.net/browse/EV2CORE-113?focusedCommentId=17006&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17006) I do not completely understand how this should be filled. 

I can put dataforms title and its properties there, but [currently](https://github.com/egovernment/eregistrations/pull/1780/files/04e110b7dfdea7ff5266a5a2befb54c9a06367f8#diff-e1390d0bdaa281d91c2f9bb546767447R220) they are being added to schema.items.properties.data property. Should I remove that?